tests/queries/saved_queries/commands_tests.py

```diff
@@ -108,3 +120,109 @@ def test_export_query_command_key_order(self, mock_g):
             "version",
             "database_uuid",
         ]
+class TestImportSavedQueriesCommand(SupersetTestCase):
+    def test_import_v1_saved_queries(self):
+        """Test that we can import a saved query"""
+        contents = {
+            "metadata.yaml": yaml.safe_dump(saved_queries_metadata_config),
+            "databases/imported_database.yaml": yaml.safe_dump(database_config),
+            "queries/imported_query.yaml": yaml.safe_dump(saved_queries_config)
+        }
+
+        command = ImportSavedQueriesCommand(contents)
+        command.run()
+
+        saved_query = db.session.query(SavedQuery).filter_by(
+            uuid=saved_queries_config["uuid"]
+        ).one()
+        assert saved_query.schema == "public"
+        assert saved_query.sql == (
+            """
+            -- Note: Unless you save your query, 
+            these tabs will NOT persist if you clear
+            your cookies or change browsers.
+
+            SELECT * from birth_names
+            """
+        )
+        assert saved_query.uuid == "05b679b5-8eaf-452c-b874-a7a774cfa4e9"
+        assert saved_query.database_uuid == "b8a1ccd3-779d-4ab7-8ad8-9ab119d7fe89"
+
+        database = (
+            db.session.query(Database).filter_by(uuid=database_config["uuid"]).one()
+        )
+
+        db.session.delete(saved_query)
+        db.session.delete(database)
+        db.session.commit()
+    def test_import_v1_saved_queries_multiple(self):
+        """Test that a saved query can be imported multiple times"""
+        contents = {
+            "metadata.yaml": yaml.safe_dump(saved_queries_metadata_config),
+            "databases/imported_database.yaml": yaml.safe_dump(database_config),
+            "queries/imported_query.yaml": yaml.safe_dump(saved_queries_config)
+        }
+        command = ImportSavedQueriesCommand(contents, overwrite=True)
+        command.run()
+        command.run()
+        database = (
+            db.session.query(SavedQuery).filter_by(uuid=database_config["uuid"]).one()
+        )
+        saved_query = db.session.query(SavedQuery).filter_by(datasource_id=database.id).all()
+        assert len(saved_query) == 1
+
+        db.session.delete(saved_query[0])
+        db.session.delete(database)
+        db.session.commit()
+        
+    def test_import_v1_saved_queries_validation(self):
+        """Test different validations applied when importing a chart"""
+        # metadata.yaml must be present
+        contents = {
+            "metadata.yaml": yaml.safe_dump(saved_queries_metadata_config),
```

Comment:
You need to remove `metadata.yaml`, since here you're testing that a validation message "Missing metadata.yaml" is shown when the file is missing.